test/test_misc.py

```diff
@@ -581,6 +582,75 @@ def kernel(fn, t: torch.Tensor):
         )
         ast.parse(code)
 
+    @skipIfPyTorchBaseVerLessThan("2.10")
```

Comment:
PyTorch 2.9 has a indexing expression related bug that's only fixed in PyTorch nightly: https://github.com/pytorch/pytorch/pull/131761. Making a patch for that in Helion framework is quite complicated, thus we skip this unit test unless we are on 2.10 version (including current nightly).